src/breedsRouter.js

```diff
@@ -1,16 +1,26 @@
+// TODO provide DEL, POST, PUT routes also. 
```

Comment:
That can be part of a separate PR - let's get the current functionality squared away first.